Application/EdFi.Ods.AdminApi/Features/Applications/ApplicationModel.cs

```diff
@@ -16,6 +16,8 @@ public class ApplicationModel
     public string? ProfileName { get; set; }
     public int EducationOrganizationId { get; set; }
     public string? OdsInstanceName { get; set; }
+    public int? VendorId { get; set; }
+    public int? ProfileId { get; set; }
```

Comment:
Can you add (or modify) an end-to-end test that proves that these two new fields are now working?